std_msgs/Header header
float32 cost
cinempc/DroneAndCameraState[] mpc_n_states
float32[] plot_values